tests/test_rabfe.py

```diff
@@ -22,14 +28,77 @@
 
 
 class TestRABFEModels(TestCase):
-    def test_predict_complex_decouple(self):
-        """Just to verify that we can handle the most basic complex decoupling RABFE prediction"""
+    def test_predict_rabfe_model(self):
```

Comment:
This test is pretty slow, 80s. The other two tests in this file are 50s and 10s, and both are now covered implicitly by this test. Moving it to slow tests doesn't seem to make an impact on the time. Not sure what to do yet, but the test seems like a good one if we are to rely on the RABFE protocol